package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0-development",
   "dependencies": {
     "babel-runtime": "^6.6.1",
-    "bowser": "1.4.6",
+    "bowser": "2.7.0",
```

Comment:
Didn't bowser do a breaking change?

Reply:
In the editor at least we had to do a change: https://github.com/livingdocsIO/livingdocs-editor/commit/d9626e8ca8e06066d9459eb43f68d546a2dce9f8

Reply:
Ah that's indeed problematic, it doesn't error but causes some problems I'll fix that